build.gradle

```diff
@@ -826,7 +819,6 @@ subprojects {
         toolVersion = versions.jacoco
       }
 
-      // NOTE: Jacoco Gradle plugin does not support "offline instrumentation" this means that classes mocked by PowerMock
```

Comment:
You can't just remove the first line, the following line does not mean anything now.

Reply:
Sorry for that. I remove related comments.